### PR TITLE
Added some sample UI to test the request function

### PR DESCRIPTION
**Pull request by binhonglee**
- Added some basic form to take input
- Fixed the request sending format to be aligned with server side

To do
- [ ] Fix the result handling (currently just throws an error and might or might not crashes the page)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,63 +1,49 @@
+/*jshint esversion: 6 */
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
-var https = require('https');
+var https = require('http');
```
Comment (binhonglee) [inline src/App.js:4]: This should be changed back once we have a proper `https` domain.
```suggestion
var https = require('https');
```

File: src/App.js [hunk 2]
```diff
@@ -66,13 +52,42 @@ class App extends Component {
...
+        callback(returnData);
+      });
+    });
+    get_req.write(JSON.stringify(data));
+    get_req.end();
+  }
+
+  requestOption(data, type) {
+    return {
+      host:  '127.0.0.1',
```
Comment (binhonglee) [inline src/App.js:81]: Same for this.
```suggestion
      host:  'api.sambal.life', 
```

File: src/App.js [hunk 3]
```diff
@@ -66,13 +52,42 @@ class App extends Component {
...
+    });
+    get_req.write(JSON.stringify(data));
+    get_req.end();
+  }
+
+  requestOption(data, type) {
+    return {
+      host:  '127.0.0.1', 
+      path: '/todo/api/v1.0/recipes/',
+      port: 5000,
```
Comment (binhonglee) [inline src/App.js:83]: And this.
```suggestion
```

**Review by evelinfong — APPROVED** [review]
LGTM

**Pull request merged**